src/java.base/share/classes/java/lang/Object.java

```diff
@@ -49,7 +50,7 @@
  * @see     java.lang.Class
  * @since   1.0
  */
-@AnnotatedFor({"guieffect", "index", "lock", "nullness"})
+@AnnotatedFor({"guieffect", "index", "lock", "nullness", "aliasing"})
```

Comment:
Let's keep the list of type systems sorted, so put "aliasing" as the first system.